Chefs/chefsRouter.js

```diff
@@ -0,0 +1,55 @@
+const router = require("express").Router();
+const DB = require("./chefsModel.js");
+
+// TO FIND A SPECIFIC CHEFS PAGE
+router.get("/:id", async (req, res) => {
+  const { id } = req.params;
+  try {
+    const chef = await DB.findChef(id);
+    if (chef) {
+      res.status(200).json(chef);
+    } else {
+      res.status(404).json({ message: "No account associated with this ID." });
+    }
+  } catch (err) {
+    res.status(400).json(err.message);
+  }
+});
+
+//TO CREATE A NEW CHEF ACCOUNT
+router.post("/", (req, res) => {
+  let chefInfo = req.body;
+
```

Comment:
You will want to check to make sure all required fields have been sent before sending a request to the database.